app/grandchallenge/profiles/validators.py

```diff
@@ -0,0 +1,17 @@
+from django.contrib.auth.validators import UnicodeUsernameValidator
```

Comment:
How did you know to include `UnicodeUsernameValidator`, are there other default validators we are now missing from the list?

Reply:
it's described in here: https://docs.djangoproject.com/en/4.1/ref/contrib/auth/#django.contrib.auth.validators.UnicodeUsernameValidator as the default validator, and we don't configure any others